docker/docker/docker-compose.yaml

```diff
@@ -0,0 +1,135 @@
+# To start docker-compose:
+# 1. Copy the example .env file:
+#    cp ../docker/webapi/.env.example ../docker/webapi/.env
+# 2. Edit the .env file to add the corresponding configuration values.
+# 3. Copy the example .env file for the web searcher plugin:
+#    cp ../docker/plugins/web-searcher/.env.example ../docker/plugins/web-searcher/.env
+# 4. Edit the .env file for the web searcher plugin to add the corresponding configuration values.
+# 5. With the .env files populated, you can now start docker-compose by running "docker-compose up --build"
+
+version: "3"
+services:
+  chat-copilot-webapp-nginx:
+    image: chat-copilot-webapp-nginx
+    build:
+      context: ..
+      dockerfile: docker/webapp/Dockerfile.nginx
+      args:
+        - REACT_APP_BACKEND_URI=http://localhost:3080
+    ports:
+      - 3000:3000
+    depends_on:
+      chat-copilot-webapi:
+        condition: service_started
+  #  chat-copilot-webapp:
+  #    image: chat-copilot-webapp
+  #    build:
+  #      context: ..
+  #      dockerfile: docker/webapp/Dockerfile
+  #    ports:
+  #      - 3000:3000
+  #    environment:
+  #      - REACT_APP_BACKEND_URI=http://localhost:3080
+  #    depends_on:
+  #      chat-copilot-webapi:
+  #        condition: service_started
+  chat-copilot-webapi:
+    image: chat-copilot-webapi
+    build:
+      context: ..
+      dockerfile: docker/webapi/Dockerfile
+    ports:
+      - 3080:8080
```

Comment:
I'm curious.  Why not 8080:8080?

Reply:
port 8080 usually will be used by other services if you are running docker-compsoe file from a developer machine, so choose 3080 to avoid any potential port conflcition issue